Makefile

```diff
@@ -12,11 +12,11 @@ fmt-docs:
 
 lint: vendor
 	@echo "✓ Linting source code with https://staticcheck.io/ ..."
-	@staticcheck ./...
+	@go run honnef.co/go/tools/cmd/staticcheck@v0.4.0 ./...
```

Comment:
We're using 0.3.2, but latest release is 0.4.3. While I'm happy to bump the version of this linter, should we do this in a separate PR?

If we do this, we should remove https://github.com/databricks/terraform-provider-databricks/blob/remove-legacy-dlvLoadConfig/.github/workflows/push.yml#L40-L41.

Reply:
done